napi/minify/test/terser.test.ts

```diff
@@ -24,13 +24,16 @@ import { run_code } from './sandbox';
 
 function run(input: string, expected: string[], prepend_code?: string) {
   const consoleMock = vi.spyOn(console, 'log').mockImplementation(() => undefined);
-  const minified = minify('test.mjs', input).code;
-  expect(minified).not.toBeFalsy();
-  // Use `consoleMock` instead of the returned output.
-  const _ = run_code(minified, prepend_code);
-  const calls = consoleMock.mock.calls.map((args) => args.map(convert).join(' '));
-  expect(calls).toStrictEqual(expected);
-  consoleMock.mockReset();
+  try {
+    const minified = minify('test.mjs', input).code;
+    expect(minified).not.toBeFalsy();
+    // Use `consoleMock` instead of the returned output.
+    const _ = run_code(minified, prepend_code);
+    const calls = consoleMock.mock.calls.map((args) => args.map(convert).join(' '));
+    expect(calls).toStrictEqual(expected);
+  } finally {
+    consoleMock.mockReset();
```

Comment:
When a test fail, all tests after that failed because `mockReset` was not called.